src/kernel.cr

```diff
@@ -320,24 +321,24 @@ end
 # sprintf "%20.8b", -11   # => "               -1011"
 # ```
 #
-# Precision for `e` is number of digits after the decimal point.
+# Precision for `e` is number of digits after the decimal point
```

Comment:
I think I'd rather add full stops where they're missing, instead of removing them here.

Reply:
@straight-shoota great. I ended up adding colons, which are used commonly to introduce code examples, and are also present in the code base.